app/scripts/migrations/160.ts

```diff
@@ -201,11 +203,25 @@ function transformState(state: Record<string, unknown>) {
 
     updatedSubjects.push(key);
 
+    const existingCaip25Caveat = getExistingCaip25PermissionCaveat(subject);
     const caip25PermissionCaveatWithCurrentChainIdSet = addPermittedEthChainId(
-      getExistingCaip25PermissionCaveat(subject),
+      existingCaip25Caveat,
       currentChainId,
     );
 
+    let caip25PermissionCaveatWithCurrentChainIdAndAccountsSet =
+      caip25PermissionCaveatWithCurrentChainIdSet;
+
+    if (existingCaip25Caveat) {
+      const alreadyPermissionedAccounts = getEthAccounts(existingCaip25Caveat);
```

Comment:
`existingPermittedAccounts`? or just plain old `permittedAccounts`? idk. nit.

Reply:
switched to `existingPermittedAccounts` here: https://github.com/MetaMask/metamask-extension/pull/33111/commits/9e3276cbbd5160a89162cc3e05af411863d20653